packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -690,4 +714,28 @@ function CenteredWrapper({children}) {
  * Tables are containers for displaying information. They allow users to quickly scan, sort, compare, and take action on large amounts of data.
  */
 const _TableView = React.forwardRef(TableView) as <T>(props: SpectrumTableProps<T> & {ref?: DOMRef<HTMLDivElement>}) => ReactElement;
-export {_TableView as TableView};
+
+/* 
+  When ready to remove this feature flag, you can remove this whole section of code, delete the _DEPRECATED files, and just replace the export with the _TableView above.
+*/
+function FeatureFlaggedTableView<T extends object>(props: SpectrumTableProps<T>, ref: DOMRef<HTMLDivElement>) {
+  let state = useTableState({
```

Comment:
Just an FYI for other reviewers, this is indeed a second copy of Table State, used purely to determine if anything is trying to allow resizing

Question @dnorth, if someone is conditionally putting allowsResizing on a column, will that be a problem? one way this could happen is if there is a column with resizing where the entire column is removed. We have a story that removes columns.

Reply:
The bug we were seeing is fixed by this commit: https://github.com/adobe/react-spectrum/pull/2883/commits/3bf4357067413227be473f50067e1a557d6abc93